README.md

```diff
@@ -40,6 +40,40 @@ The following command will restart all pods in the `web` and `jobs` deployments:
 
 `kubernetes-restart <kube namespace> <kube context> --deployments=web,jobs`
 
+### Deploying Kubernetes secrets
+
```

Comment:
Did we want to emphasize to only use this when actual K8S secrets are needed, and to stick with normal ejson workflow otherwise?